### PR TITLE
3536: Add create org confirmation page

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/neworg/routes.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    res.redirect(`${config.baseUrl}/organisations/find/select/create/confirmation`);
+  }));
+
+  router.get('/select/create/confirmation', authProvider.authorise(), withCatch(authProvider, async (req, res) => {
+    logger.info('navigating to create organisation confirmation page');
+    const context = await getCreateOrgConfirmationContext();
+    res.render('common/pages/confirmation.njk', addContext({ context, user: req.user }));
+  }));
+
+  return router;
```
Comment (alibombaye) [inline app/pages/neworg/routes.js:50]: oooh I like this. Good idea!

File: app/pages/neworg/routes.test.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    .post(postPath)
+    .type('form')
+    .set('Cookie', [cookies, mockUnauthorisedCookie])
+    .send({ _csrf: csrfToken })
+    .expect(200)
+    .then((res) => {
+      expect(res.text.includes('data-test-id="error-page-title"')).toEqual(true);
+      expect(res.text.includes('Not authorised')).toEqual(true);
+    });
+};
```
Comment (alibombaye) [inline app/pages/neworg/routes.test.js:109]: 🙄 have we got these routes test helper functions in 2 places now?

Comment (kat1906) [reply app/pages/neworg/routes.test.js:109]: yeah I will tidy them up tomorrow when doing the rest of the tidy up

File: app/pages/neworg/routes.test.js [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+      return request(setUpFakeApp())
+        .get(path)
+        .set('Cookie', [mockAuthorisedCookie])
+        .expect(200)
+        .then((res) => {
+          expect(res.text.includes('data-test-id="create-org-page"')).toEqual(true);
+          expect(res.text.includes('data-test-id="error-page-title"')).toEqual(false);
+        });
+    });
+  });
```
Comment (alibombaye) [inline app/pages/neworg/routes.test.js:253]: take it you are going to do the tests POST /organisations/find/select/create and the GET /select/create/confirmation at a later date?

Comment (kat1906) [reply app/pages/neworg/routes.test.js:253]: oops forgot - added them

**Pull request merged**